pkg/daemon/daemon.go

```diff
@@ -637,31 +635,44 @@ func (dn *Daemon) restartDevicePluginPod() error {
 	return nil
 }
 
-func (dn *Daemon) loadVendorPlugins(ns *sriovnetworkv1.SriovNetworkNodeState) error {
-	var pl []string
+func (dn *Daemon) EnableVendorPlugins(ns *sriovnetworkv1.SriovNetworkNodeState) error {
```

Comment:
by no mean a must but you could move these two methods (EnableVendorPlugins and registerVendorPlugins) to daemon/plugins.go

Reply:
much cleaner agree, done